packages/elements/src/components/ino-nav-drawer/ino-nav-drawer.tsx

```diff
@@ -67,6 +67,10 @@ export class NavDrawer implements ComponentInterface {
       this.drawerInstance.open = this.open || false;
     }
 
+    this.el.shadowRoot
```

Comment:
Instead of querying for the element, I would suggest to use it's reference like this:
```jsx
<aside class={classDrawer} ref={(el) => this.drawerEl = el}>
```

Reply:
Done ✅ I also directly attached the event listener for the `click` event to the `ino-icon-button`.